src/NzbDrone.Core/ImportLists/Custom/CustomSettings.cs

```diff
@@ -0,0 +1,33 @@
+using FluentValidation;
+
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.ImportLists.Custom
+{
+    public class CustomSettingsValidator : AbstractValidator<CustomSettings>
+    {
+        public CustomSettingsValidator()
+        {
+            RuleFor(c => c.BaseUrl).ValidRootUrl();
+        }
+    }
+
+    public class CustomSettings : IImportListSettings
+    {
+        private static readonly CustomSettingsValidator Validator = new CustomSettingsValidator();
+
+        public CustomSettings()
+        {
+            BaseUrl = "";
+        }
+
+        [FieldDefinition(0, Label = "List URL", HelpText = "The URL for the show list")]
```

Comment:
What type of show list? What format does it need to be in?

Reply:
Maybe create a documentation page for it, and link to it?

Reply:
```json
[
  {
    "tvdbId": "75837"
  },
  {
    "tvdbId": "77847"
  },
  {
    "tvdbId": "78299"
  },
  {
    "tvdbId": "72756"
  }
]
```

Thats whats required